### PR TITLE
DAOS-9173 ofi: revert fi_cancel patch

### PR DESCRIPTION
**Pull request by johannlombardi**
Revert libfabric patch due to issues with soak.

Signed-off-by: Johann Lombardi <johann.lombardi@intel.com>

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -158,7 +164,7 @@ libfabric (1.3.0-4) unstable; urgency=medium
 libfabric (1.3.0-3) unstable; urgency=medium
 
   * Update watch file. Upstream uses github only now.
-  * Install libfabric.pc. Thanks Marcin Ślusarz. (Closes: #826222)
+  * Install libfabric.pc. Thanks Marcin Slusarz. (Closes: #826222)
```
Comment (brianjmurrell) [inline debian/changelog:167]: Is this change intentional?

**Review by brianjmurrell — APPROVED** [review]
Looks good otherwise.

File: debian/changelog (hunk 1 above)
Comment (johannlombardi) [reply debian/changelog:167]: nope
